packages/orchestrator-ui-components/src/components/WfoProcessList/WfoProcessesList.tsx

```diff
@@ -244,13 +244,9 @@ export const WfoProcessesList = ({
     return (
         <WfoTableWithFilter<ProcessListItem>
             queryString={queryString}
-            data={
-                !isFetching
-                    ? mapGraphQlProcessListResultToProcessListItems(
-                          processes || [],
-                      )
-                    : []
-            }
+            data={mapGraphQlProcessListResultToProcessListItems(
+                processes || [],
+            )}
```

Comment:
`WfoSubscriptionsList` does it slightly different, not sure if that matters?

```
data={
                data
                    ? mapGraphQlSubscriptionsResultToSubscriptionListItems(data)
                    : []
            }
```

Reply:
Effectively it is the same, but will update that one too